documentation/PATh/htc_workloads/using_data/data-transfer.md

```diff
@@ -0,0 +1,160 @@
+---
+path:
+  path: htc_workloads/using_data/data-transfer.md
+---
+
+# Data Staging and Transfer to Jobs
+
+Table of Contents: 
+[TOC]
+
+# Overview
+
+As a distributed system, jobs in the PATh Facility can run in different physical  locations, where the computers that are executing jobs don't have direct access to the files placed on the Access Point (e.g. in a home directory on ap1.facility.path-cc.io). In order to run on this kind of distributed system, jobs need to "bring along" the data, code, packages, and other files from the Access Point (where the job is submitted) to the PATh Facility execute points (where the job will run).  HTCondor's file transfer tools and plugins make this possible; input and output files are specified as part of the job submission and then moved to and from the execution location. 
+
+This guide describes where to place files on PATh Facility Access Points, and how to use these files within jobs. 
+
+# Transferring Data To/From HTCondor Jobs
+
+There are two spaces for placing files on the PATh Facility Access Point, and each has a corresponding transfer method for referencing files in the submit file. 
+
+<table>
```

Comment:
I think we should simplify these tables. It is too easy to get into the "I have 1 50MB file, and one 500MB - what do I do?". My preference would be something like: <4GB total use CEDAR, >4GB total use stash

Reply:
I think those numbers have to be lower - 500MB instead of 4GB. 
Otherwise, I think that's okay?

Reply:
1GB